packages/react-storybook/src/client/index.js

```diff
@@ -7,5 +8,15 @@ export const configure = previewApi.configure;
 export const getStorybook = previewApi.getStorybook;
 
 // NOTE export these to keep backwards compatibility
-export { action } from '@kadira/storybook-addon-actions';
-export { linkTo } from '@kadira/storybook-addon-links';
+import { action as _action } from '@kadira/storybook-addon-actions';
+import { linkTo as _linkTo } from '@kadira/storybook-addon-links';
+
+export const action = deprecate(
+  _action,
```

Comment:
This dangling underscore will become an eslint error in the future, can you pick another naming pattern?